index.html

```diff
@@ -487,6 +482,7 @@
 </html>
 
 <!-- preserve before running Tidy -->
+<!--
```

Comment:
Yes, it defines the markdown anchor links: https://daringfireball.net/projects/markdown/syntax#link. We frequently reference the same link/concept in multiple places.. This allows us to define the link once for the concept and then wrap all the concepts in []'s that autolink to the right place.